version/openjdk8.sh

```diff
@@ -5,5 +5,5 @@ sudo yum install -y java-1.8.0-openjdk-devel
 
 ln -s /usr/lib/jvm/java-1.8.0-openjdk /usr/lib/jvm/java-8-openjdk-amd64
 
-update-alternatives --install /usr/bin/java java /usr/lib/jvm/java-8-openjdk-amd64/jre/bin/java
-update-alternatives --install /usr/bin/javac javac /usr/lib/jvm/java-8-openjdk-amd64/jre/bin/javac
+sudo update-alternatives --install /usr/bin/java java /usr/lib/jvm/java-8-openjdk-amd64/bin/java 1
```

Comment:
why does this need sudo? are all commands not running as `root` already?

Reply:
i will remove that